frontend/src/app/about/FutureFunctionality.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from './page.module.css'
+
+export default function FutureFunctionality () {
+    return (
+        <section className={styles.section}>
+          <h2 className={styles.subheading}>Future functionality</h2>
+          <p>
+            In the future we would like to add further functionality to the
+            website.
+          </p>
+          <p>Including the following features:</p>
+          <ul className={styles.futures_features_list}>
+            <li>Users can log in</li>
```

Comment:
Forgive me if I’ve missed this but with the user being able to login, would that mean they would have their own profile? If so I would say something like  “Add authentication and user profile”

Reply:
Updated to new wording